UnityProject/Assets/Scripts/Items/Weapons/ExplosiveBase.cs

```diff
@@ -26,7 +26,7 @@ public class ExplosiveBase : SignalReceiver
 		[Header("Explosive settings")]
 		[SerializeField] protected ExplosiveType explosiveType;
 		[SerializeField] protected bool detonateImmediatelyOnSignal;
-		[SerializeField] protected int timeToDetonate = 10;
+		[SerializeField, SyncVar] protected int timeToDetonate = 10;
```

Comment:
I thought you could only change SyncVar from server side?

Reply:
NetUI does the change server sided

Reply:
I'm confused since in the description you say it was only being done client side ? 
so that means it surely isn't communicating with the server?

Reply:
oH, you're right. Yeah something doesn't add up here and I noticed that the cooldown function is only called on the server and has the `[Server]` attribute above it. something is odd here.

Reply:
Yeah I don't know what exactly the issue is then because everything seems to be fine prefab and code wise, going to wait until next playtest until we get this happening again.